src/App.tsx

```diff
@@ -27,9 +27,9 @@ function App() {
 
   return (
     <div className="App">
-      {/* <OrderService> */}
-      <HomePage />
-      {/* </OrderService> */}
+      <OrderService>
+        <HomePage />
+      </OrderService>
     </div>
```

Comment:
Why we include de OrderService like this, and the other services are in the useEffect?